test/controllers/isValidTimestamp.js

```diff
@@ -0,0 +1,45 @@
+/*global describe:false, it:false, beforeEach:false, afterEach:false, request:false, mock:false*/
+
+'use strict';
+
+var mongoose = require('mongoose');
+var mockgoose = require('mockgoose');
+
+describe('/isValidTimestamp', function() {
```

Comment:
These tests are testing the service logic and don't belong here in the controller test.